### ISSUE TITLE
Greasing considerations to endpoint features

### ISSUE THREAD

**Issue by chris-wood** [intarchboard/draft-protocol-greasing#3]
In my mind, greasing is currently defined with respect to _protocol features_, with something like the QUIC spin bit being one obvious example. However, the concepts and motivations behind greasing can also extend to endpoint features themselves, such as deciding whether or not to use QUIC at all in the first place. The [Privacy Pass](https://ietf-wg-privacypass.github.io/base-drafts/draft-ietf-privacypass-auth-scheme.html#name-user-interaction) specification applies this pattern, wherein client implementations can choose to provide tokens when asked or not (according to some local grease implementation behavior). If we believe that greasing extends to features in this way, should the document talk about it, and if so, what should it say?

**Comment by LPardue** [comment]
@chris-wood @tfpauly maybe this idea extends to e.g. happy eyeballs v3?

**Comment by LPardue** [comment]
Seems to me we could fit some text in https://intarchboard.github.io/draft-protocol-greasing/draft-edm-protocol-greasing.html#section-4 so would happily review a proposal

**Comment by tfpauly** [comment]
Yeah the privacy pass example works well for section 4, and I think we could talk about happy eyeballs-like client-side algorithms changing their timing and rules to make sure networks don't ossify around particular behaviors.
